draft-ietf-quic-recovery.md

```diff
@@ -135,6 +135,12 @@ Crypto Packets:
 : Packets containing CRYPTO data sent in Initial or Handshake
   packets.
 
+Out-of-order Packets:
+
+: Packets that do not increase the largest received packet number for its
+packet number space by exactly one. Packets usually arrive out of order
```

Comment:
```suggestion
  packet number space by exactly one. Packets arrive out of order
```

Reply:
The exceptions I had in mind are packet duplication and misbehaving senders.

Reply:
Duplication doesn't count as out-of-order because the packet is ignored.  Misbehaving senders are certainly possible, but I'm not sure how important they are to this text?